Makefile

```diff
@@ -143,6 +143,18 @@ oc-create-worker: oc-create-configmap oc-create-secrets
 		-p SOURCE_REPOSITORY_REF=$(shell git rev-parse --abbrev-ref HEAD) \
 	| oc create -f -
 
+oc-create-scheduler: oc-create-configmap oc-create-secrets
```

Comment:
I haven't tried `oc-create-all` in awhile so I'm not entirely sure if it works or not but, you might as well add this to it.